docker-entrypoint.sh

```diff
@@ -65,7 +65,7 @@ fi
 # sub-directory, so we can use the presence of some file/dir as a marker to
 # detect that mounting was a success. Execute the command on success.
 
-su - $RUN_AS -c "s3fs $DEBUG_OPTS ${S3FS_ARGS} \
+su - $RUN_AS -c "s3fs $DEBUG_OPTS -o ${S3FS_ARGS} \
```

Comment:
What happens when `S3FS_ARGS` is not set?

Reply:
Thank you for the hint! Next commit handles this case in a better way, I guess.